src/dither.cpp

```diff
@@ -57,10 +59,12 @@ double Dither::update()
 
   // Generates gaussian random noise using the polar method.
   double v1, v2, r;
+  // uniform distribution on the interval [0.0, 1.0]
+  std::uniform_real_distribution<double> distribution(0.0, std::nextafter(1.0, std::numeric_limits<double>::max()));
```

Comment:
Does it make sense to use [-1 ,1] right away?

Reply:
good point, change to [-1, 1]